src/pipelines/mod.rs

```diff
@@ -1,5 +1,5 @@
-mod copydir;
-mod copyfile;
+mod copy_dir;
+mod copy_file;
```

Comment:
**Pro tip:** any changes not core to the purpose of your PR should be reverted. These changes are def not core. HOWEVER, just so that I do not dissuade you from future contribution, let's keep this. Not a big deal, is fairly well isolated, and is easy enough to verify.

Reply:
Just so I understand this correctly: You don't want me to revert the renaming of those two modules, correct?

Reply:
Correct, let's keep it for now. My point is just that changes like this (which are unnecessary and purely a matter of preference) can make review a bit more difficult.